package.json

```diff
@@ -36,9 +36,8 @@
     "react-redux": "^7.2.0",
     "react-select": "^3.1.0",
     "react-shadow": "^17.6.0",
-    "react-spatial": "^0.3.7",
+    "react-spatial": "0.4.0-beta.1",
```

Comment:
if we merge on master can you use a non-beta release pls?

Reply:
✅